ui/app/pages/error/error.component.js

```diff
@@ -29,17 +29,30 @@ class ErrorPage extends PureComponent {
     );
   }
 
+  renderHelpLink() {
+    return (
+      <a
+        target="_blank"
+        rel="noopener noreferrer"
+        href="https://metamask.zendesk.com/hc/en-us/requests/new"
+      >
+        <span className="error-page__link-text">{this.context.t('here')}</span>
+      </a>
+    );
+  }
+
   render() {
     const { error } = this.props;
     const { t } = this.context;
 
     const isPopup = getEnvironmentType() === ENVIRONMENT_TYPE_POPUP;
+    const messageKey = isPopup ? 'errorPagePopupMessage' : 'errorPageMessage';
 
     return (
       <section className="error-page">
         <h1 className="error-page__header">{t('errorPageTitle')}</h1>
         <h2 className="error-page__subheader">
-          {isPopup ? t('errorPagePopupMessage') : t('errorPageMessage')}
+          {t(messageKey, [this.renderHelpLink()])}
```

Comment:
[REQ] I don't see the harm in rendering the <a> tag inline here instead of extracting to a render function. We won't have this option of separating out parts of our JSX without creating intermediary components in files that follow our new paradigms (functional components).

Reply:
Makes sense, force of habit was at work here :)

Reply:
Done